index.html

```diff
@@ -518,6 +563,19 @@ <h6>Importance</h6>
 
 						---
 
+						### Other elements of interest
+						We've covered the basics to get you started, but there are many more elements available, some of which are very semantically useful:
+
+						- section
+						- article
+						- footer
+						- em
+						- strong
+
+						Check out a [complete list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) for more info.
```

Comment:
While some semantic elements have been slipped into the example code, they're not generally called out. I think pointing out the good stuff is important to teaching students _modern_ web development, as opposed to the slightly-outdated GDI curriculum.

Reply:
This is also something covered in Class II but this teaser would be a great addition!